code_schemes/gender.json

```diff
@@ -14,27 +14,27 @@
       "Shortcut": "e"
     },
     {
-      "StringValue": "male",
+      "StringValue": "man",
       "Shortcut": "m",
-      "DisplayText": "male",
+      "DisplayText": "man",
       "VisibleInCoda": true,
       "NumericValue": 1,
       "CodeID": "code-63dcde9a",
       "CodeType": "Normal",
       "MatchValues": [
-        "male"
+        "man"
```

Comment:
I think changing this will break the auto-coders in the pipeline if you do this, because core data still uses the terms male and female. If we want to fix it there, probably better to set the MatchValues here to ["man", "male"] for now, then update in core.

Reply:
Thanks updated here - will share the core pr next